### ISSUE TITLE
Insufficient fee on Bitcoin testnet

### ISSUE THREAD

**Issue by rkfg** [BoltzExchange/boltz-lnd#44]
I'm trying to do a reverse swap on Bitcoin testnet but can't claim the on-chain coins. Here's the log:
```
INFO : 2021/06/05 11:00:18 Paid invoice of Reverse Swap m1Ipaf with fee of 0.000 satoshis
INFO : 2021/06/05 11:00:20 Reverse Swap m1Ipaf status update: transaction.mempool
INFO : 2021/06/05 11:00:20 Constructing claim transaction for Reverse Swap m1Ipaf with output: 1fd891c868c6781fed76afab2bbe87eaeef864281ae68c5b5d5e1ea6cc8e8638:0
INFO : 2021/06/05 11:00:20 Using fee of 0 sat/vbyte for claim transaction
INFO : 2021/06/05 11:00:20 Constructed claim transaction: ab7394756904bb02d04f2902cb5f4f69586e2f110fb00d18c0303a531283e725
ERROR: 2021/06/05 11:00:20 Could not finalize claim transaction: could not broadcast transaction: min relay fee not met, 0 < 136
INFO : 2021/06/05 11:03:04 Reverse Swap m1Ipaf status update: transaction.confirmed
INFO : 2021/06/05 11:03:04 Constructing claim transaction for Reverse Swap m1Ipaf with output: 1fd891c868c6781fed76afab2bbe87eaeef864281ae68c5b5d5e1ea6cc8e8638:0
INFO : 2021/06/05 11:03:04 Using fee of 0 sat/vbyte for claim transaction
INFO : 2021/06/05 11:03:04 Constructed claim transaction: ab7394756904bb02d04f2902cb5f4f69586e2f110fb00d18c0303a531283e725
ERROR: 2021/06/05 11:03:04 Could not finalize claim transaction: could not broadcast transaction: min relay fee not met, 0 < 136
```

I don't see a way to specify fee in `boltzd` and `boltzcli`.

**Comment by michael1011** [comment]
The fee is fetched from our API. Seems like our testnet API was down at the time of your swap :thinking:
That shouldn't have happened in the first place, so I'll investigate that. And I think that adding a fallback for the fee estimation from the API would make sense too. Let's say try fetching a fee from the API and if that fails, take the fee estimation from LND

Thank you for reporting the issue @rkfg!

**Comment by michael1011** [comment]
Closed by #50
